### PR TITLE
Added Norwegian translation.

### PR DESCRIPTION
**Pull request by runarsf**

### CODE REVIEW

File: README.no.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+- [Nederlands](README.nl.md)
+- [简体中文](README.zh_hans.md)
+- [繁體中文](README.zh_hant.md)
+- [Türkçe](README.tr.md)
+- [Bangla](README.bn.md)
+- [Русский](README.ru.md)
+- [Polski](README.pl.md)
+- [Esperanto](README.eo.md)
+- [Svenska](README.se.md)
+- [Norsk](README.no.md)
```
Comment (yuta0801) [inline README.no.md]: no need all links (see #37)

Comment (runarsf) [reply README.no.md]: Fixed.

**Comment by yuta0801** [comment]
Please add the link in README.md

**Comment by runarsf** [comment]
> 
> 
> Please add the link in README.md

Link added.

**Pull request merged**